pasion_preciouskaye/script.js

```diff
@@ -0,0 +1,62 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const commentForm = document.getElementById('comment');
+    const commentsList = document.getElementById('comments-list');
+    const sortAscButton = document.getElementById('sort-asc');
+    const sortDescButton = document.getElementById('sort-desc');
+    const nameInput = document.getElementById('name');
+    const commentInput = document.getElementById('ycomment');
+    const submitButton = document.getElementById('button_comment');
+  
+    commentForm.addEventListener('submit', function (event) {
+      event.preventDefault();
+      addComment();
+    });
+  
+    sortAscButton.addEventListener('click', function () {
+      sortComments('asc');
+    });
+  
+    sortDescButton.addEventListener('click', function () {
+      sortComments('desc');
+    });
+  
+    nameInput.addEventListener('input', checkFormValidity);
+    commentInput.addEventListener('input', checkFormValidity);
+  
+    function checkFormValidity() {
+      if (nameInput.value.trim() !== '' && commentInput.value.trim() !== '') {
+        submitButton.disabled = false;
+      } else {
+        submitButton.disabled = true;
+      }
+    }
```

Comment:
Kindly review this code as the submit button is still disabled even if the name and comment have inputs already.

If possible, I think you can have this code written in guard clause. 

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR04rIg_SBkft7ED5muFsxm9FUkijQq-aweBdgT21-tlg-UpXKtfkO4Nb9s_aem_AanCGQNw3v8o-ZJK1bprYBSLNcABosgmI-0dnlGiU_GSgrlxM4drKiB_S99jcEItxjMmHfHXZ64dF_L3NbTavA6s#slide=id.g25d183194fb_0_0